### DISCUSSION TITLE
Testing dataframe values in first assignment do not pass

### DISCUSSION THREAD

**Discussion by misterskye** [earthlab-education/Earth-Analytics-2023-01-Intro#207] [Q&A02 Open Reproducible Science] [answered]
For some unknow reason when I'm working thru the Max-Temp_over-time... module I keep getting stopped at where I create the dataframe.  That code brings up the dataframe and by first glance looks good.
![image](https://github.com/earthlab-education/Earth-Analytics-2023-01-Intro/assets/47870472/e5dcca14-b6ef-48d6-b485-fbb007b0c4b8)

But, when I run the Test Cell code I get the 'Oops- your data are not correct'.  How can this be?  All I have completed is to bring the data into the dataframe.  
At a loss.

**Comment by advythr** [comment]
Hi Skye,

I am having the same problem but I'm not sure how to fix it.

<img width="1667" alt="Screen Shot 2023-09-11 at 2 20 10 PM" src="https://github.com/earthlab-education/Earth-Analytics-2023-01-Intro/assets/83184164/3b704707-e070-43c5-9742-446d09ede0af">

And the test cell says "Oops - your data are not correct."
<img width="419" alt="Screen Shot 2023-09-11 at 2 20 56 PM" src="https://github.com/earthlab-education/Earth-Analytics-2023-01-Intro/assets/83184164/600bfd7b-4439-464f-a68c-6b2f064c0e85">

My suspicion is that this is occurring because the values in the data we downloaded (which seem to be the same) are different from the values for the same dataset in the tutorial video. It looks like the test cell is assessing whether we got the right data using summary statistics for the dataset.

```
summary = [round(val, 2) for val in tmax_df_resp.mean().values]
if summary == [198362.0, 61.93]:
    points += 6
    print('\u2705 Great work! You correctly downloaded data.')
else:
    print('\u274C Oops - your data are not correct.')
```

I'm thinking perhaps the dataset was updated recently with new values (perhaps corrections) and that's why the test cell no longer works?

@eculler Could this be the reason?

**Reply by misterskye** [reply]
Agreed--- I tried changing the temp to the 1983 value ( 62.7), but I'm unsure of the 62.0 value in 198362.0.

Needless to say, it didn't work

**Comment by eculler** [comment, marked as answer]
@ea-students You're right, my test values didn't get updated with a recent change. So all the 'values' tests will be wrong, but I will still give you credit. Not sure how that got through, since I believe they were passing when I made my video...but sometimes these things happen! **Please don't change the test cells**, since that will make it difficult for me to run your notebooks when I grade!

If you would like to check your mean values, the correct ones are:

Test 1: should pass
Test 2: [198362.0, 62.05]
Test 3: [1983.5, 62.05]
Test 4: [1983.5, 62.05, 16.69]

The are computed with the following code, which computes the mean of all columns and then rounds it to two decimal points: print([round(val, 2) for val in temp_df.mean().values])
